src/routing/approved.rs

```diff
@@ -2085,27 +2110,26 @@ impl Approved {
             return Err(Error::InvalidDstLocation);
         }
 
+        let variant = Variant::UserMessage(content);
+
         // If the msg is to be aggregated at dst, we don't vote among our peers, wemsimply send the msg as our vote to the dst.
         let msg = if itinerary.aggregate_at_dst() {
             Message::for_dst_accumulation(
-                &self.node,
                 self.section_keys_provider.key_share()?,
+                self.section().prefix().name(),
```

Comment:
why not use `itinerary.src.name()` here?

Reply:
It was that before, but I think your suggestion makes more sense. I'll change it.